### PR TITLE
[Spanish] localize 'Idempotence'

### LINKED ISSUE #1376: [Spanish] localize 'Idempotence'
Localize the `content/en/idempotence.md` document to Spanish

### PR DESCRIPTION
**Pull request by tinniaru3005**
### Describe your changes
- Converted `idempotence.md` to Spanish
- Added the new words to .wordlist.txt

### Related issue number or link (ex: `resolves #issue-number`)
resolves #1376 
Closes #1376 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/es/idempotence.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: Idempotence
+status: Terminado
+category: Propiedad
+tags: ["infraestructura", "arquitectura", ""]
```
Comment (electrocucaracha) [inline content/es/idempotence.md:5]:
```suggestion
status: Completed
category: Propiedad
tags: ["infrastructure", "architecture", ""]
```

File: content/es/idempotence.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+---
+
+En matemáticas o informática, la idempotencia describe una operación que siempre conduce al mismo resultado,
+no importa cuantas veces lo ejecutes.
```
Comment (electrocucaracha) [inline content/es/idempotence.md:9]:
```suggestion
no importa cuantas veces se ejecute.
```

File: content/es/idempotence.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+---
+title: Idempotence
```
Comment (electrocucaracha) [inline content/es/idempotence.md:2]:
```suggestion
title: Idempotencia
```

File: content/es/.wordlist.txt [hunk 4]
```diff
@@ -7,6 +7,7 @@ API
 APIs
 application
 Appreciated
+arquitectura
```
Comment (electrocucaracha) [inline content/es/.wordlist.txt:10]:
```suggestion
```

File: content/es/.wordlist.txt [hunk 5]
```diff
@@ -91,7 +92,9 @@ hipervisor
 https
 IaaS
 IaC
+Idempotence
```
Comment (electrocucaracha) [inline content/es/.wordlist.txt:95]:
```suggestion
```

File: content/es/.wordlist.txt [hunk 6]
```diff
@@ -91,7 +92,9 @@ hipervisor
...
 iamnoah
+infraestructura
```
Comment (electrocucaracha) [inline content/es/.wordlist.txt:97]:
```suggestion
```

**Review by electrocucaracha — CHANGES_REQUESTED** [review]
These translation PRs require targeting to `dev-es` branch. Please, change them accordingly.

**Comment by tinniaru3005** [comment]
Hey @electrocucaracha. Can you please tell me if it is fine now?

File: content/es/.wordlist.txt [hunk 7]
```diff
@@ -131,6 +131,7 @@ Not
 OCI
 of
 on
+operación
 on-premises
 Open
 Paganini
```
Comment (electrocucaracha) [inline content/es/.wordlist.txt:137]:
```suggestion
on-premises
Open
operación
Paganini
```

**Review by electrocucaracha — CHANGES_REQUESTED** [review]
The translation looks good, but the word list is unsorted. This PR also requires to be recreated using `dev-es` as branch base.

**Comment by electrocucaracha** [comment]
I sent you a [PR](https://github.com/tinniaru3005/cncf-glossary/pull/1) with the only commit, need it for this. I think if you merge it in your branch, this will fix it.

**Comment by tinniaru3005** [comment]
> I sent you a [PR](https://github.com/tinniaru3005/cncf-glossary/pull/1) with the only commit, need it for this. I think if you merge it in your branch, this will fix it.

Hey @electrocucaracha, I did it. Thank you so much for the help. :)

**Comment by electrocucaracha** [comment]
@raelga @krol3 @92nqb could you take a look this PR? It seems to be ready for review.

**Review by 92nqb — APPROVED** [review]

**Pull request merged**